React/Fabric/Mounting/ComponentViews/Modal/RCTModalHostViewComponentView.mm

```diff
@@ -180,6 +183,28 @@ - (void)ensurePresentedOnlyIfNeeded
   }
 }
 
+- (void)handleDetent:(ModalHostViewDetent)detent
+{
+  if (@available(iOS 15.0, *)) {
+    if (self.viewController.sheetPresentationController) {
+      NSArray<UISheetPresentationControllerDetent *> *detents = @[UISheetPresentationControllerDetent.largeDetent];
+      switch (detent) {
+        case ModalHostViewDetent::Large:
+          break;
+        case ModalHostViewDetent::Medium:
+          detents = @[UISheetPresentationControllerDetent.mediumDetent];
+          break;
+        case ModalHostViewDetent::MediumResizable:
+              self.viewController.sheetPresentationController.prefersGrabberVisible = true;
+              self.viewController.sheetPresentationController.largestUndimmedDetentIdentifier =  UISheetPresentationControllerDetentIdentifierMedium;
+          detents = @[UISheetPresentationControllerDetent.mediumDetent, UISheetPresentationControllerDetent.largeDetent];
```

Comment:
this is misaligned, it will trigger a linting error for sure.

Reply:
I see, fixed it :)